catalogue_pipeline/terraform/iam_role_policy.tf

```diff
@@ -15,6 +15,11 @@ resource "aws_iam_role_policy" "transformer_task_cloudwatch_metric" {
   policy = "${data.aws_iam_policy_document.allow_cloudwatch_push_metrics.json}"
 }
 
+resource "aws_iam_role_policy" "transformer_s3_messages" {
+  role   = "${module.transformer.task_role_name}"
+  policy = "${data.aws_iam_policy_document.allow_s3_messages_put_get.json}"
```

Comment:
Is this correct? Doesn't the transformer just need Put permissions, and the ID minter just need Get permissions?